client/capstone-app/src/index.js

```diff
@@ -4,12 +4,23 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import {BrowserRouter} from "react-router-dom"
+import { Auth0Provider } from "@auth0/auth0-react";
+
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   // <React.StrictMode>
+  
     <BrowserRouter>
-    <App/>
+      <Auth0Provider
+    domain="dev-oc5sv8wa1mdvaybx.us.auth0.com"
```

Comment:
Add `clientId` and `domain` in `.env`, it's sensitive details that should not be available on github.